fuzz/fuzz_targets/editing.rs

```diff
@@ -63,12 +87,15 @@ fuzz_target!(|data: (StartingText, Vec<EditOp>)| {
                 while !rope.is_char_boundary(byte_range.end) {
                     byte_range.end += 1;
                 }
-                rope.replace(byte_range, text);
+                rope.replace(byte_range.clone(), text);
+                string.replace_range(byte_range, text);
             },
 
             _ => continue,
         }
     }
 
     rope.assert_invariants();
+    assert_eq!(string, chunks_to_string(rope.byte_slice(..rope.byte_len())));
```

Comment:
`Rope` implements `PartialEq<String>`, so this can be replaced by:

```suggestion
    assert_eq!(rope, string);
```

Reply:
I specifically wanted to check that by concatenating the chunks we get the whole content; IMO `PartialEq<String>` check can be added as well, but probably should not replace the chunk-based check (event if the `PartialEq` uses chunks internally).